src/Commands/EnvObliterateCommand.php

```diff
@@ -51,14 +51,14 @@ public function obliterate($site_name)
         $this->providerManager()->validateCredentials();
 
         // Do nothing without confirmation
-        if (!$this->confirm('Are you sure you want to delete {site} AND its corresponding GitHub repository {url} and CircleCI configuration?', ['site' => $site->getName(), 'url' => $url])) {
+        if (!$this->confirm('Are you sure you want to delete {site} AND its corresponding GitHub repository {url} and CI configuration?', ['site' => $site->getName(), 'url' => $url])) {
```

Comment:
If we are changing CircleCI to just CI should we also change GitHub to Git provider?

Reply:
Good idea